.travis.yml

```diff
@@ -29,8 +29,13 @@ after_success:
 
 deploy:
 - provider: script
-  skip_cleanup: true
+  cleanup: false
   script: curl -sL http://git.io/goreleaser | bash
   on:
     tags: true
     condition: $TRAVIS_OS_NAME = linux
+- provider: script
+  script: $HOME/gopath/bin/goveralls -coverprofile=v2/coverage.out -service travis-ci
+  on:
+    tags: true
+    condition: $TRAVIS_OS_NAME = linux && $TRAVIS_GO_VERSION =~ ^1.12.
```

Comment:
Newline maybe? Also, should you update the top and here to a more recent version of Go?

Reply:
fixed